integration/util_test.go

```diff
@@ -483,6 +486,32 @@ networks:
 `, nw)
 	}
 
+	zotConfigFileStr := fmt.Sprintf(`
```

Comment:
nit: Could we move this up to where the other configs are declared at the top of the file and use `testutil.ApplyTextTemplate` to fill in the host address portion